### PR TITLE
[Feature Tweak] Message Fixy For Snowballs/Kickstands

### PR DESCRIPTION
**Pull request by TheFurryFeline**
Tweaks output of messages for snowball destroying/compacting and kickstand interactions. 

Would like some advice about the visible_message stuff since that's where it was going wrong in the first place and gave weird messages to everyone. Seen it do "You compact the snowball" to everyone. Same for the kickstands.

Changelog Notes:

- Tweaks bike kickstand usage message and snowball compacting/crushing message to give the name of its user rather than a public weirdness of "You put down/up the kickstand" or "You smash the snowball" or "You start compacting the snowball in your hand".

### CODE REVIEW

File: code/game/objects/items/weapons/material/misc.dm [hunk 1]
```diff
@@ -71,14 +71,16 @@
 	w_class = ITEMSIZE_SMALL
 	attack_verb = list("mushed", "splatted", "splooshed", "splushed") // Words that totally exist.
 
-/obj/item/weapon/material/snow/snowball/attack_self(mob/user as mob)
+/obj/item/weapon/material/snow/snowball/attack_self(mob/user as mob)	//TFF 22/33/20 - changes to to_chat line to be show only to self. ToDo: proper fix for the thing.
```
Comment (Atermonera) [inline code/game/objects/items/weapons/material/misc.dm]: ???

Comment (TheFurryFeline) [reply code/game/objects/items/weapons/material/misc.dm]: Ehhhh, I s'pose I could remove that considering that the visible_message isn't working right there. Wanted to have a reminder to return to it and figure out _why_ it hasn't been working the way it should.

**Comment by TheFurryFeline** [comment]
C'mon, Travis.

**Comment by Atermonera** [comment]
Well GITHUB thinks the checks haven't completed but the CHECKS say that they HAVE completed successfully so this should probably be fine?

**Pull request merged**